internal/command/command_test.go

```diff
@@ -1020,13 +1045,13 @@ func fakeRegistryHandler(resp http.ResponseWriter, req *http.Request) {
 			resp.Header().Set("Content-Type", "application/json")
 			resp.WriteHeader(200)
 			if movedNamespace, ok := movedProviderNamespaces[name]; ok {
-				resp.Write([]byte(fmt.Sprintf(`{"id":"%s/%s","moved_to":"%s/%s","versions":[{"version":"1.0.0","protocols":["4"]}]}`, namespace, name, movedNamespace, name)))
+				fmt.Fprintf(resp, `{"id":"%s/%s","moved_to":"%s/%s","versions":[{"version":"1.0.0","protocols":["4"]}]}`, namespace, name, movedNamespace, name)
```

Comment:
Weird that this is not producing another linting issue 🤔 .
[fmt.Fprintf](https://pkg.go.dev/fmt#Fprintf) returns number of bytes and error which are unchecked here.

Reply:
Good catch! fixed in fb8afbff57

Reply:
This is part of golang-ci-lint's auto-rewriting and must have internal rules about this.